lambdas/indexer/app.py

```diff
@@ -484,3 +484,39 @@ def nudge(event: chalice.app.CloudWatchEvent):
         log.info('Nudging queue with %i token(s) for a total value of %i', len(tokens), num_documents)
         for batch in chunked(tokens, 10):
             _send_tokens(batch)
+
+
+@app.schedule('rate(5 minutes)', name=config.failure_message_lambda_basename)
+def retrieve_failure_messages(_event: chalice.app.CloudWatchEvent):
+    """
+    Get all the messages from the fail queue and move these messages to the the DynamoDB failure message table.
+    """
+    database = boto3.resource('dynamodb')
+    table = database.Table(config.dynamo_failure_message_table_name)
+    with table.batch_writer() as writer:
+        fail_queue = queue(config.fail_queue_name)
+        while True:
+            received_messages = fail_queue.receive_messages(AttributeNames=['All'],
+                                                            MaxNumberOfMessages=10,
+                                                            WaitTimeSeconds=20,
+                                                            VisibilityTimeout=300)
+            if received_messages:
+                batch_requests = []
+                for message in received_messages:
+                    body = json.loads(message.body)
+                    writer.put_item(Item={
+                        'MessageType': 'bundle_notification' if 'notification' in body.keys() else 'other',
+                        'SentTimeMessageId': f"{message.attributes['SentTimestamp']}-{message.message_id}",
```

Comment:
```suggestion
                        'SentTimeMessageId': message.attributes['SentTimestamp'] + '-' + message.message_id,
```

Looks less dense due to white space.